### PR TITLE
code-format: Use ambv/black

### PR DESCRIPTION
**Pull request by aexvir**
Closes https://github.com/kiwicom/crane/issues/15

### CODE REVIEW

File: crane/cli.py [hunk 1]
```diff
@@ -13,56 +13,165 @@ def strip_trailing_slash(_, param, value):
...
-@click.option('--sleep-after-upgrade', envvar='CRANE_SLEEP_AFTER_UPGRADE', default=0, help='seconds to wait after upgrade', show_default=True)
-@click.option('--manual-finish', envvar='CRANE_MANUAL_FINISH', default=False, is_flag=True, help='skip automatic upgrade finish')
-@click.option('--slack-token', envvar='CRANE_SLACK_TOKEN', default=None, help='Slack API token')
-@click.option('--slack-channel', envvar='CRANE_SLACK_CHANNEL', default=None, help='Slack channel to announce in')
-@click.option('--slack-link', envvar='CRANE_SLACK_LINK', multiple=True, type=(str, str), metavar='TITLE URL', help='links to mention in Slack')
-@click.option('--sentry-webhook', envvar='CRANE_SENTRY_WEBHOOK', default=None, help='Sentry release webhook URL', callback=strip_trailing_slash)
-@click.option('--webhook-url', envvar='CRANE_WEBHOOK_URL', default=None, multiple=True, help='URLs to POST the release status to', callback=strip_trailing_slash)
-@click.option('--webhook-token', envvar='CRANE_WEBHOOK_TOKEN', default=None, help='auth token for webhooks')
-@click.option('--datadog-key', envvar='CRANE_DATADOG_KEY', default=None, help='key for posting release events')
```
Comment (underyx) [inline crane/cli.py]: Can we exclude this maybe?

Comment (aexvir) [reply crane/cli.py]: Mmm sure

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+[tool.black]
+exclude = 'crane/cli.py'
```
[the remark this answers is not in the document]
Comment (aexvir) [reply pyproject.toml]: Could have been more explicit 🙃 I'll change it now

[the remark this answers is not in the document]
Comment (underyx) [reply pyproject.toml]: Yup, sorry, that's my bad 😇

**Pull request merged**